apps/src/applab/ChartApi.js

```diff
@@ -4,327 +4,284 @@
  * For now, uses Google charts.
  * @see {GoogleChart}
  */
-/* global Promise */
 import * as utils from '../utils';
```

Comment:
Nit: it looks like only `utils.quote` is ever used? 

```suggestion
 import {quote} from '../utils';
```